### PR TITLE
Add `--skip-tables=<tables>` argument to exclude tables when performing search-replace

### PR DESCRIPTION
**Pull request by marksabbath**
### Issue #31 
- Implemented a method to exclude tables by adding a new argument `--skip-tables`

### CODE REVIEW

**Comment by marksabbath** [comment]
Added functional tests @danielbachhuber as you've requested.

**Comment by danielbachhuber** [comment]
@marksabbath Can you merge master for #49?

**Comment by marksabbath** [comment]
Done by https://github.com/wp-cli/search-replace-command/pull/48/commits/282570efbbaf61baeba158704c1fff15d9cec8c6 @danielbachhuber

File: src/Search_Replace_Command.php [hunk 1]
```diff
@@ -269,8 +275,16 @@ public function __invoke( $args, $assoc_args ) {
 
 		// Get table names based on leftover $args or supplied $assoc_args
 		$tables = \WP_CLI\Utils\wp_get_table_names( $args, $assoc_args );
+
+		// Removes from $tables tables provided by skip-tables argument
+		$tables = array_diff( $tables, $this->skip_tables );
+
 		foreach ( $tables as $table ) {
 
+			if ( in_array( $table, $this->skip_tables ) ) {
```
Comment (gitlost) [inline src/Search_Replace_Command.php]: Just to note that this wasn't removed when the `array_diff()` was added and made it redundant.

Comment (danielbachhuber) [reply src/Search_Replace_Command.php]: Good spot, #50

**Pull request merged**